core/src/main/java/io/undertow/server/protocol/framed/AbstractFramedStreamSinkChannel.java

```diff
@@ -73,8 +73,8 @@ public abstract class AbstractFramedStreamSinkChannel<C extends AbstractFramedCh
 
 
     static {
-        final int defaultAwaitWritableTimeout = 600000;
-        int await_writable_timeout = AccessController.doPrivileged((PrivilegedAction<Integer>) () -> Integer.getInteger("io.undertow.await_writable_timeout", defaultAwaitWritableTimeout));
+        final long defaultAwaitWritableTimeout = 600000000000L;
```

Comment:
This is not readable, lets create sustainable software, e.g. replace with:

    final long defaultAwaitWritableTimeout = TimeUnit.MINUTES.toNanos(10);

Reply:
You are right, I fixed it. 
Also, I am unsure about reading `io.undertow.await_writable_timeout` property in nanoseconds. I would change it to seconds or minutes as well, and then parse it to nanoseconds. But I can't find where this property can be set.